AndelaOSP/wire
# Add tooltip to assignee on incident cards

[Delivers #158622967]

#### What does this PR do?
Adds tooltips on assignee icons to show full names on hover
#### How should this be manually tested?
Run the application and hover over the assignee on the incident card to see the tooltip
#### What are the relevant pivotal tracker stories?
[#158622967](https://www.pivotaltracker.com/story/show/158622967)
#### Screenshots (if appropriate)
<img width="543" alt="screen shot 2018-06-27 at 15 00 55" src="https://user-images.githubusercontent.com/19855216/41972710-50d339b4-7a1b-11e8-9780-d153db9a8883.png">
